### DISCUSSION TITLE
Implement the data-wp-if directive

### DISCUSSION THREAD

**Discussion by giuliodelmastro** [WordPress/gutenberg#65956] [Interactivity API]
Hi guys.

It would be useful to be able to have a directive like “x-if” in AlpineJS, where the element is rendered only if the condition in the attribute is true.

You can use “data-wp-bind--hidden” hides only the element in the dom rendering it anyway.

Also, “data-wp-bind” does not support comparison operations and wants the boolean value directly. e.g: 

❌ data-wp-bind--hidden="context.day < 1”  

A directive like “data-wp-if” would make it much easier to develop components, even the most complex ones.

**Comment by luisherranz** [comment]
Closing this discussion to centralize the conversation in this other one:

- https://github.com/WordPress/gutenberg/discussions/53058
